node/src/neighborhood/mod.rs

```diff
@@ -3733,68 +3855,38 @@ mod tests {
                     FallbackPreference::ExitCountryWithFallback
                 );
                 assert_eq!(
-                    neighborhood
-                        .neighborhood_database
-                        .node_by_key(&cz_public_key)
-                        .unwrap()
-                        .metadata
-                        .country_undesirability,
+                    neighborhood.get_node_country_undesirability(&cz_public_key),
                     UNREACHABLE_COUNTRY_PENALTY,
-                    "cz We expecting {}, country is too close to be exit",
+                    "cz We expect {}, country is too close to be exit",
```

Comment:
Ja vim, ze tohle vypada uz moc hnidopissky, ale kdybys chtel:

Muzes opravit ty hlasky v tech assertions. 
Spise neco jako: "CZ: we expect" nebo "We expect CZ with {}, country...".